client/post-editor/editor-sidebar/index.jsx

```diff
@@ -44,6 +47,8 @@ export class EditorSidebar extends Component {
 					onSave={ onSave }
 					confirmationSidebarStatus={ confirmationSidebarStatus }
 				/>
+				{ isEnabled( 'gutenberg' ) && <EditorGutenbergOptInSidebar /> }
```

Comment:
We might want to pick a different feature flag so we don't need to update e2e tests for now. (We've enabled `gutenberg` on wpcalypso, and canary tests run in that env.)